static/js/src/openstack/react/utils/variables.js

```diff
@@ -0,0 +1,56 @@
+const TCO_VARIABLES = {
```

Comment:
Since they don't change, shouldn't it be TCO_CONSTANTS?

Reply:
makes sense 😅